roxie/ccd/ccdfile.cpp

```diff
@@ -1128,11 +1128,20 @@ class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress
         {
             CriticalBlock b(crit);
             Linked<ILazyFileIO> f = files.getValue(localLocation);
-            if (f && f->isAlive())
+            while (f && f->isAlive())
```

Comment:
continue isn't breaking out of the loop, it's going round it again... I need to recheck the size/date etc match too, so I'd be repeating three lines of code in your version I think...